.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 Cargo.lock
-/.idea
\ No newline at end of file
+/.idea
+.vscode
```

Comment:
This change doesn't need to be part of the PR

Reply:
Yeah I didn't notice, it was automatic